setup.py

```diff
@@ -21,15 +21,22 @@
 ]
 
 install_requires=[
-    'stanford-stk @ git+https://github.com/stanford-futuredata/stk.git@main',
-    'grouped_gemm @ git+https://github.com/tgale96/grouped_gemm@main',
-    'mosaicml-turbo==0.0.4',
+    "triton==2.1.0",
```

Comment:
does this need strict pin? Might cause problems down the road if someone else (eg torch) starts upgrading.... 

Could we lower bound?

Reply:
I'm concerned about Triton breaking us with a newer release. We could make it >=2.1.0, but that's a little risky.